.circleci/config.yml

```diff
@@ -334,3 +350,35 @@ jobs:
       - persist_to_workspace:
           root: ~/
           paths: project/bin
+
+  build-package:
+    executor: golang
+    parameters:
+      package_type:
+        type: enum
+        enum: ["deb", "rpm"]
+    steps:
+      - checkout
+      - attach_to_workspace
+      - run:
+          name: Install fpm and dependencies
+          command: |
+            sudo apt-get update
```

Comment:
should an `upgrade` happen here as well?

Reply:
`upgrade` upgrades other deps, update just refreshes the repositories.